submit-api/src/Cardano/TxSubmit/Node.hs

```diff
@@ -21,105 +21,42 @@ module Cardano.TxSubmit.Node
 import Cardano.Prelude hiding
```

Comment:
In my next PR, where I'll be adding support for both the Shelley and Cardano (HFC) protocols, most of the contents of this module, `Cardano.TxSubmit.Node`, will be deleted. 

Instead of maintaining a long-running session with the local node, we'll instead be using `Cardano.Api.TxSubmit.submitTx` from `cardano-api`.